modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -460,26 +504,7 @@ def convert_utc_to_local_time(date, tz)
         end
       end
 
-      # Returns the facility timezone id (eg. 'America/New_York') associated with facility id (location_id)
-      def get_facility_timezone_memoized(facility_location_id)
```

Comment:
I made both `get_facility_timezone_momoized` and `get_facility` public as they're used outside of the service. Lmk if there are any considerations I'm missing by doing that

Reply:
I don't think you necessarily need to do this. I think ideally the fields that use these functions would be returned in the appts list. For example, `get_facility_timezone_memoized` is always used to set `facility_timezone` but if that data was just available in the appts list returned from the service then the downstream controllers and serializers wouldn't need access to `get_facility_timezone_memoized`. Similar logic goes for `get_facility`. That being said I'm fine with doing this for now and making a new ticket to follow up with the concept above if `get_facility` is fairly complex

Reply:
Hmm do you think there's any concern with automatically doing another upstream fetch? Is there any scenario where we'd want the appointment info separate from facility info? I'm happy to make that change if you'd rather just have it done in this pr

Reply:
Let's just leave it as is, but please make another ticket to follow up with the engineers from the appts team about baking this into the service response because I think at the end of all of this migration stuff the ideal situation is that the controllers (web and mobiles) are doing little to nothing other than serializing the data received from the service